docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,227 @@
+
+Details on what constitutes a valid staking transaction can be found 
+[here](./staking-script.md#staking-transaction).
+
+`unbonding_tx`: 
+An unbonding transaction allows a BTC staker to unlock their funds by 
+obtaining necessary signatures and broadcasting the transaction to the 
+Bitcoin network. Once confirmed, the staker regains access to their BTC.
+
+The unbonding transaction is in hex format. This transaction is used to 
+unlock staked BTC after the unbonding period. You can generate this transaction 
+using our Golang library or TypeScript library, see 
+[Babylon staker library](https://github.com/babylonlabs-io/btc-staker/blob/main/staker/types.go) or 
+[TypeScript library](https://github.com/babylonlabs-io/simple-staking/blob/main/src/app/hooks/services/useTransactionService.ts). Details on what constitutes a valid 
+unbonding transaction can be found [here](./staking-script.md#unbonding-transaction).
+
+`slashing_tx`: 
+The slashing transaction is in hex format. Generated during both staking and 
+unbonding phases, this transaction spends the staking or unbonding transaction 
+in cases of protocol violations such as double-signing. To see exactly what 
+constitutes a valid slashing transaction, see the 
+[`Slashing Transaction` section](./staking-script.md#slashing-transaction).
+A valid signature for this transaction must be included in the 
+`slashing_tx_sig` field.
+
+`delegator_slashing_sig`: 
+The signature for the `slashing_tx`. This must be a valid signature 
+generated using the private key corresponding to the BTC public key (`btc_pk`).
+
+`delegator_unbonding_slashing_sig`: 
+The unbonding slashing signature for the delegator. This must be a valid 
+signature generated using the private key corresponding to the BTC public key 
+(`btc_pk`).
+
+`pop` (Proof of Possession):
+A cryptographic signature verifying ownership of the Bitcoin private key used 
+for staking. Required for registering a phase-1 stake to confirm the legitimate 
+ownership of the associated Bitcoin address. This will be generated during the 
+phase-1 creation of stake process. To see the full message, see the 
+[`ProofOfPossessionBTC` message](./proto/babylon/btcstaking/v1/pop.proto).
+
+`inclusion_proof` (Optional): 
+A Merkle proof showing that the staking transaction is included in the 
+Bitcoin chain. This field is optional but recommended. To see the full message, 
+see the [`InclusionProof` message](./proto/babylon/btcstaking/v1/btcstaking.proto).
+
+If not provided during submission, it must be submitted later using the 
+`MsgAddBTCDelegationInclusionProof` message. To see the full message, see the 
+[`MsgAddBTCDelegationInclusionProof` message](./proto/babylon/btcstaking/v1/tx.proto).
+
+`Other General Fields`:
+- `staking_value`: The amount of BTC locked for delegation (in satoshis).
+- `staking_time`: The timelock period for the staking transaction (in blocks).
+- `unbonding_time`: The timelock period for unbonding (in blocks).
+
```

Comment:
We should detail here on ways the `MsgCreateBTCDelegation` can be constructed and communicated to Babylon. Some options:
- CLI
- Creation using typescript with ref here: https://github.com/babylonlabs-io/simple-staking/blob/main/src/app/hooks/services/useTransactionService.ts (please sync with Jerome on the precise line and commit to use) and broadcast to baby node
- Creation using golang library: https://github.com/babylonlabs-io/babylon/blob/main/x/btcstaking/types/tx.pb.go and broadcast to baby node